integration/networking/bridge_linux_test.go

```diff
@@ -216,70 +213,8 @@ func TestBridgeICC(t *testing.T) {
 	}
 }
 
-// TestBridgeICCWindows tries to ping container ctr1 from container ctr2 using its hostname.
-// Checks DNS resolution, and whether containers can communicate with each other.
-// Regression test for https://github.com/moby/moby/issues/47370
-func TestBridgeICCWindows(t *testing.T) {
-	skip.If(t, testEnv.DaemonInfo.OSType != "windows")
-
-	ctx := setupTest(t)
-	c := testEnv.APIClient()
-
-	testcases := []struct {
-		name    string
-		netName string
-	}{
-		{
-			name:    "Default nat network",
-			netName: "nat",
-		},
-		{
-			name:    "User defined nat network",
-			netName: "mynat",
-		},
-	}
-
-	for _, tc := range testcases {
-		t.Run(tc.name, func(t *testing.T) {
-			ctx := testutil.StartSpan(ctx, t)
-
-			if tc.netName != "nat" {
-				network.CreateNoError(ctx, t, c, tc.netName,
-					network.WithDriver("nat"),
-				)
-				defer network.RemoveNoError(ctx, t, c, tc.netName)
-			}
-
-			const ctr1Name = "ctr1"
-			id1 := container.Run(ctx, t, c,
-				container.WithName(ctr1Name),
-				container.WithNetworkMode(tc.netName),
-			)
-			defer c.ContainerRemove(ctx, id1, containertypes.RemoveOptions{Force: true})
-
-			pingCmd := []string{"ping", "-n", "1", "-w", "3000", ctr1Name}
-
-			const ctr2Name = "ctr2"
-			attachCtx, cancel := context.WithTimeout(ctx, 5*time.Second)
-			defer cancel()
-			res := container.RunAttach(attachCtx, t, c,
-				container.WithName(ctr2Name),
-				container.WithCmd(pingCmd...),
-				container.WithNetworkMode(tc.netName),
-			)
-			defer c.ContainerRemove(ctx, res.ContainerID, containertypes.RemoveOptions{Force: true})
-
-			assert.Check(t, is.Equal(res.ExitCode, 0))
-			assert.Check(t, is.Equal(res.Stderr.Len(), 0))
-			assert.Check(t, is.Contains(res.Stdout.String(), "Sent = 1, Received = 1, Lost = 0"))
-		})
-	}
-}
-
 // TestBridgeINC makes sure two containers on two different bridge networks can't communicate with each other.
 func TestBridgeINC(t *testing.T) {
-	skip.If(t, testEnv.DaemonInfo.OSType == "windows")
```

Comment:
So, I know various tests were not put in platform specific files, but skipped  because they were not necessary platform-specific, just not yet rewritten to make them work on Windows; from the description of the test, it seems like testing that networking being properly sandboxed should be something that's also the case on Windows; so we know why we had to skip?

Or will there be a new, equivalent test in the Windows file?

Reply:
I think that's the only test that should currently have a Windows equivalent but doesn't. We'll need to add more when we get IPv6 on Windows ...

- Should be tested on both ...
  - `TestBridgeINC`
- IPv6 tests that we'll need on both when we have IPv6 support on Windows (depending on what we get working)...
  - `TestDefaultBridgeIPv6`
  - `TestDefaultBridgeAddresses`
  - `TestDisableIPv6Addrs`
  - `TestNonIPv6Network`
  - `TestNoIP6Tables`
  - `TestDisableNAT`
  - `TestProxy4To6`
- Linux-specific ...
  - `TestInternalNwConnectivity` (_--internal doesn't work on Windows_)
  - `TestSetInterfaceSysctl`
  - `TestReadOnlySlashProc`
  - `TestSetEndpointSysctl`
- Tested on both ...
  - `TestBridgeICC`
  - `TestPortMappedHairpin`

I haven't discovered a way to run integration tests locally on Windows, so writing them is a bit of a nightmare. I don't want to get into it as part of this change - so I've raised https://github.com/moby/moby/issues/48249.